bolt-design-system/bolt
# Image | FE | Focal point selection: add valign demo

## Jira

http://vjira2:8080/browse/BDS-816

## Summary

Add demo of background image `valign` option which gives users some control over the focal point of background images.

## Details

This still doesn't change anything in IE11 where object-position is not supported. IE11 users may still see an image with a confusing or awkward focal point ([see ticket](http://vjira2:8080/browse/BDS-1428.)). Is this something we plan to address?

## How to test

- View background valign variations page either on [staging](https://boltdesignsystem-5exea8lrb.now.sh/pattern-lab/patterns/02-components-background-35-background-valign-variations/02-components-background-35-background-valign-variations.html) or [locally](http://localhost:3000/pattern-lab/?p=components-background-valign-variations) on the `feature/add-background-image-valign-demo` branch.
- Review each option to verify background image is aligned properly (will be more noticeable at larger screen-sizes).
- Please weigh in on IE11 issue. 🙏
